zebra-utils/src/bin/zebra-checkpoints/main.rs

```diff
@@ -102,3 +103,9 @@ fn main() -> Result<()> {
 
     Ok(())
 }
+
+fn init_tracing_backup() {
```

Comment:
I'd just call this `init_tracing()`, the backup in the original function is because its an alternate code path for when the normal tracing setup is disabled. In this case there is no alternative tracing subscriber.

Reply:
My mistake, fixed at https://github.com/ZcashFoundation/zebra/pull/740/commits/1097c34be6c9bb28ed49e6b851bd64e094307dff